frontend/src/graph/operators/query.gql

```diff
@@ -1638,6 +1638,18 @@ query GetGitHubIntegrationSettings($workspace_id: ID!) {
 	}
 }
 
+query GetGitlabIntegrationSettings($workspace_id: ID!) {
+	is_integrated: is_workspace_integrated_with(
+		integration_type: Gitlab
+		workspace_id: $workspace_id
+	)
+	gitlab_projects(workspace_id: $workspace_id) {
+		name
+		id
+		nameWithNameSpace
```

Comment:
Gitlab projects may be organized under different namespaces. I used this on UI so that multiple 
projects with the same name may be distinguished from UI.

This might just be "overkill" depending on our expectations of "usual" interactions with this integration.
It does however cover a special case that I thought was worth "covering"